FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Tool/appsettings.json

```diff
@@ -30,16 +30,6 @@
     "requiredTypes": "",
     "typeFilters": ""
   },
-  "arrow": {
-    "readOptions": {
-      "useThreads": true,
-      "blockSize": 31457280
```

Comment:
Also help to delete the "arrow__readOptions__blockSize" environment variable in DICOM ContainerApps ARM Template?

Reply:
Sure, I will remove it. does Dicom use this variable in the code? will it occur issues if I just delete it?

Reply:
No. DICOM data is larger than FHIR, so we use "arrow__readOptions__blockSize" in ARM template to expand the size limit. No additional code use this.

Reply:
OK, removed.